common-types/src/main/protobuf/com/improving/app/common/domain/memberId.proto

```diff
@@ -3,6 +3,10 @@ syntax = "proto3";
 package com.improving.app.common.domain;
 import "scalapb/scalapb.proto";
 
+option (scalapb.options) = {
+  preserve_unknown_fields: false
+};
+
```

Comment:
same comment as in adress.proto

Reply:
I'd image that memberId will be extremely stable even in future iterations. For that reason, I will keep this.